test/Microsoft.ML.IntegrationTests/Training.cs

```diff
@@ -22,7 +23,7 @@ public Training(ITestOutputHelper output) : base(output)
         /// <summary>
         /// Training: It is easy to compare trainer evaluations on the same dataset.
         /// </summary>
-        [Fact]
+        [FieldAwareFactorizationMachineFact("FieldAwareFactorizationMachine doesn't currently support non x86/x64. Github isue #5871")]
```

Comment:
Lets use full links
`[https://github.com/dotnet/machinelearning/issues/5871`](https://github.com/dotnet/machinelearning/issues/5871`)

Reply:
It feels like a comment like this should go into the `FieldAwareFactorizationMachineFactAttribute` code instead of on all the tests.

Reply:
Comment has been moved inside the `FieldAwareFactorizationMachineFactAttribute `